waku/v2/protocol/waku_noise/noise.nim

```diff
@@ -98,10 +261,288 @@ proc genKeyPair*(rng: var BrHmacDrbgContext): KeyPair =
   keyPair.publicKey = keyPair.privateKey.public()
   return keyPair
 
+# Prints Handshake Patterns using Noise pattern layout
+proc print*(self: HandshakePattern) 
```

Comment:
Out of curiosity, what is the benefit of having a `stdout` print utility rather than encoding this in a `'$'` stringify procedure, which can be used for more than just stdout? Guessing it may be too complex?